src/main/c/ssl.c

```diff
@@ -1178,11 +1178,11 @@ TCN_IMPLEMENT_CALL(jint, SSL, getLastErrorNumber)(TCN_STDARGS) {
 }
 
 static void ssl_info_callback(const SSL *ssl, int where, int ret) {
-    tcn_ssl_ctxt_t *c;
+    int *handshakeCount;
     if (0 != (where & SSL_CB_HANDSHAKE_START)) {
-        c = SSL_get_app_data2(ssl);
-        if (c != NULL) {
-            ++c->handshakeCount;
+        handshakeCount = (int*) SSL_get_app_data3(ssl);
+        if (handshakeCount != NULL) {
```

Comment:
@Scottmitch btw do you prefer `if (handshakeCount != NULL)` or `if (handshakeCount)` ?